### PR TITLE
[ADD][8.0] website_event_share

### PR DESCRIPTION
**Pull request by andhit-r**
This module will add functionality to enable/disabled share button on each event details. This module will be useful if user would like to have share button on all event details. Without this module share button have to be added manually via share button snippet one by one.

Share button will appear after event location

### CODE REVIEW

File: website_event_share/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Event Share
+=========
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: This line shoudl be the same length as the title, and please put another one above the title

File: website_event_share/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Usage
+=====
+
+1. Go to website menu
+2. Open one of your event
+3. Click on customize button
+4. Toggle share button option to enable/disable share button
+
+NB: even if you have to go to one specific event, the set up is valid for all of them.
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Include try me on runbot button

File: website_event_share/README.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Share event configuration valid for all events. No individual configuration for each event
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/website/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/website/issues/new?body=module:%20website_event_share%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Change project reference

**Comment by andhit-r** [comment]
@pedrobaeza this PR is ready to be reviewed

File: website_event_share/README.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+4. Toggle share button option to enable/disable share button
+
+NB: even if you have to go to one specific event, the set up is valid for all of them.
+
+Know issues / Roadmap
+====================
+* Share event configuration valid for all events. No individual configuration for each event
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline website_event_share/README.rst:22]: This is inside section `Usage`

File: website_event_share/README.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Go to website menu
+2. Open one of your event
+3. Click on customize button
+4. Toggle share button option to enable/disable share button
+
+NB: even if you have to go to one specific event, the set up is valid for all of them.
+
+Know issues / Roadmap
+====================
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Please put the same length in the title and the equals

**Comment by andhit-r** [comment]
@rafaelbn would you kindly put **need review** label on this PR. Thanks.

**Comment by andhit-r** [comment]
Hi @OCA/crm-sales-marketing-maintainers would you kindly review this PR. Thanks in advance

**Comment by rafaelbn** [comment]
Rebuilding runbot for testing!

cc @yajo @antespi

**Comment by andhit-r** [comment]
Hi @OCA/crm-sales-marketing-maintainers would you kindly review this PR. Thanks in advance

File: website_event_share/README.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Event Share
```
Comment (elicoidal) [inline website_event_share/README.rst:6]: please add a line before the title (same length)

**Comment by elicoidal** [comment]
:+1: no test

**Comment by andhit-r** [comment]
@elicoidal thanks for your review. Revision already pushed according your latest preview

File: website_event_share/README.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: This line is one char less

File: website_event_share/README.rst [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+4. Toggle share button option to enable/disable share button
+
+NB: even if you have to go to one specific event, the set up is valid for all of them.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/199/8.0
+
+Know issues / Roadmap
+====================
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Add one more `=` (the same length as the title)

File: website_event_share/README.rst [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+* Share event configuration valid for all events. No individual configuration for each event
+
+
+Bug Tracker
+==========
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Add one more `=` (the same length as the title)

File: website_event_share/README.rst [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/event/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/event/issues/new?body=module:%20website_event_share%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+======
```
Comment (pedrobaeza) [inline website_event_share/README.rst]: Add one more `=` (the same length as the title)

**Comment by pedrobaeza** [comment]
Please squash all commits in one, as they are minimal

**Comment by andhit-r** [comment]
@pedrobaeza is it ok if i use -f to push squash commit? Or it's better to create another PR?

**Comment by pedrobaeza** [comment]
You can use without problem `push -f` to make the changes here, and thus the same PR serves.

**Comment by andhit-r** [comment]
@pedrobaeza thanks for the info. Rebase + squash.

File: website_event_share/views/website_event.xml [hunk 11]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+    <template id="event_share" name="Share Button" inherit_id="website_event.event_details" active="False" customize_show="True">
+        <xpath expr="//h4[@t-field='event.address_id']" position="after">
+            <div class="oe_share s_share text-center">
+                  <h3>Share
```
Comment (yajo) [inline website_event_share/views/website_event.xml]: Please put the "Share" word inside a `<span>` (for inheritance) and put the whole `<div>` inside a independent template.

This way I can overwrite this template to place the share buttons in another point of the event page while still being able to `t-call` the share snippet itself.

Thanks!

**Comment by andhit-r** [comment]
Hi @Yajo . I addressed your review on my latest push. Would you kindly review it again. Many thanks

**Comment by andhit-r** [comment]
Hi @OCA/crm-sales-marketing-maintainers would you kindly review this PR. Many thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
Please @antespi review, thanks

**Comment by pedrobaeza** [comment]
I have already merged it, so no need to review

**Pull request merged**